src/content/get-started/fwe/layout.md

```diff
@@ -12,98 +12,522 @@ next:
+for instance, it tries to honor that and 
+be that particular size.
+
+Others, for example [Row][] and [Column][] (flex boxes) 
+vary based on the constraints they are given. 
+Read more about flex boxes and constraints in
+the [Understanding Constraints article][].
+
+## Lay out a single widget
+
+To lay out a single widget in Flutter, 
+wrap a visible widget, 
+such as `Text` or `Image` with a widget that 
+can change its position on a screen, 
+such as a `Center` widget.
+
+:::note Note
+The examples on the page use a widget called 
+`BorderedImage`. This is a custom widget, 
+and is used here to hide
+the code that isn't relevant to this topic.
+:::
+
+```dart
+Widget build(BuildContext context) {
+  return Center(
+    child: BorderedImage(),
+  );
+}
+```
+
+The following figure shows a widget that isn't 
+aligned on the left, 
+and a widget that has been centered on the right.
+
+<img src='/assets/images/docs/fwe/layout/center.png' alt="A screenshot of a centered widget and a screenshot of a widget that hasn't been centered.">
+
+All layout widgets have either of the following:
+* A `child` property if they take a single 
+child—for example, `Center`, `Container`,
+or `Padding`.
+* A `children` property if they take a list 
+of widgets—for example, 
+`Row`, `Column`, `ListView`, or `Stack`.
+
+### Container
+
+`Container` is a convenience widget that's 
+made up of several widgets responsible for layout,
+painting, positioning, and sizing. 
+In regard to layout, 
+it can be used to add padding and 
+margin to a widget. 
+There is also a `Padding` widget
+that could be used here to the same effect. 
+The following example uses a `Container`.
+
+```dart
+Widget build(BuildContext context) {
+  return Container(
+    padding: EdgeInsets.all(16.0),
+    child: BorderedImage(),
+  );
+}
+```
+
+The following figure shows a widget without 
+padding on the left, 
+and a widget with padding on the right.
+
+<img src='/assets/images/docs/fwe/layout/padding.png' alt="A screenshot of a widget with padding and a screenshot of a widget without padding.">
+
+To create more complex layouts in Flutter, 
+you can compose many widgets. 
+For example, you can combine `Container` and `Center`:
+
+```dart
+Widget build(BuildContext context) {
+  return Center(
+    Container(
+      padding: EdgeInsets.all(16.0),
+      child: BorderedImage(),
+    ),
+  );
+}
+```
+
+## Layout multiple widgets vertically or horizontally
+
+One of the most common layout patterns is to 
+arrange widgets vertically or horizontally. 
+You can use a `Row` widget to arrange widgets 
+horizontally, 
+and a `Column` widget to arrange widgets vertically. 
+The first figure on this page used both.
+
+This is the most basic example of using a `Row` widget.
+
+{% render docs/code-and-image.md, 
+image:"fwe/layout/row.png", 
+caption: "This figure shows a row widget with three children."
+alt: "A screenshot of a row widget with three children"
```

Comment:
It's a template that puts the code and the image side by side. The alternative would be to use HTML in the file, but I figured it'd be better to make a template since I'm repeating the process so often.